CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 This file is used to list changes made in each version of grafana.
 
-### Unreleased (2019-04-25)
+### Unreleased
```

Comment:
small thing, but when you release, I think it looks better in the super market if you remove the unreleased section - and then add it back on the next PR